imports/client/components/HuntEditPage.tsx

```diff
@@ -0,0 +1,558 @@
+                  guildId={guildId}
+                  disable={disableForm}
+                  value={puzzleHooksDiscordChannel}
+                  onChange={onPuzzleHooksDiscordChannelChanged}
+                />
+                <FormText>
+                  If this field is specified, when a puzzle in this hunt is added or solved, a
+                  message will be sent to the specified channel.
+                </FormText>
+              </Col>
+            </FormGroup>
+
+            <FormGroup as={Row}>
+              <FormLabel column xs={3} htmlFor="hunt-form-firehose-discord-channel">
+                Firehose Discord channel
+              </FormLabel>
+              <Col xs={9}>
+                <DiscordChannelSelector
+                  id="hunt-form-firehose-discord-channel"
+                  guildId={guildId}
+                  disable={disableForm}
+                  value={firehoseDiscordChannel}
+                  onChange={onFirehoseDiscordChannelChanged}
+                />
+                <FormText>
+                  If this field is specified, all chat messages written in puzzles associated with
+                  this hunt will be mirrored to the specified Discord channel.
+                </FormText>
+              </Col>
+            </FormGroup>
+
+            <FormGroup as={Row}>
+              <FormLabel column xs={3} htmlFor="hunt-form-member-discord-role">
+                Discord role for members
+              </FormLabel>
+              <Col xs={9}>
+                <DiscordRoleSelector
+                  id="hunt-form-member-discord-role"
+                  guildId={guildId}
+                  disable={disableForm}
+                  value={memberDiscordRole}
+                  onChange={onMemberDiscordRoleChanged}
+                />
+                <FormText>
+                  If set, then members of the hunt that have linked their Discord profile are added
+                  to the specified Discord role. Note that for continuity, if this setting is
+                  changed, Jolly Roger will not touch the old role (e.g. to remove members)
+                </FormText>
+              </Col>
+            </FormGroup>
+          </>
+        ) : (
+          <Alert variant="info">
+            <FontAwesomeIcon icon={faInfo} fixedWidth />
+            Discord has not been configured, so Discord settings are disabled.
+          </Alert>
+        )}
+
+        <div ref={footer}>
+          {submitState === SubmitState.FAILED && <Alert variant="danger">{errorMessage}</Alert>}
+          {submitState === SubmitState.SUCCESS && (
+            <Alert variant="success" dismissible onClose={onSuccessDismiss}>
+              Hunt information successfully updated
+            </Alert>
+          )}
+
+          <Button variant="secondary" type="button" onClick={goBack}>Go back</Button>
+          {' '}
+          <Button variant="primary" type="submit" disabled={disableForm}>Save</Button>
```

Comment:
Can we make this button align to the right edge of the form?  I think "bottom right" is where I see a majority of "submit" buttons in forms in general.

(And if there were multiple buttons, the default "do a useful thing" action should be the one aligned with the edge of the form, with other actions growing out towards the left as needed)

Reply:
On reflection: I notice we're pretty inconsistent with "forms have save buttons on the right edge", and probably a big part of that is that the button is easy to lose on the far right if the page is very wide.  As a thought for the future: we might be well-served by picking a reasonable `max-width` to apply to some of our other forms and then moving the buttons to the right edge for them too.  It'd certainly help a bit with the visual balance; we tend to be very left-heavy at present (I'm thinking especially of OwnProfilePage and SetupPage).

Reply:
Yeah, +1 - I'll wait until we land #767 and then incorporate its action button class.